src/main/resources/resources/logisim/strings/gui/gui_pt.properties

```diff
@@ -19,47 +19,47 @@ revertAppearanceAction = Restaurar forma
 showStateDialogCancelButton = Cancelar
 showStateDialogEmptyNode = Circuito vazio %s
 showStateDialogNodeTitle = Estado para %s
-showStateDialogOkButton = ESTÁ BEM.
+showStateDialogOkButton = Ok.
```

Comment:
Is `Ok` fine or common form is rather `OK`?

Reply:
Ok is fine in that case, the common form sounds a bit misleading in that specific case.

Reply:
I meant capitalization. The most common form is `OK` (both letters capitalized). In fact we should have one form across whole translation. If you say `Ok` is fine (small `k`) and `OK` (capitalized  `K` is not) - could you please ensure we have all `OK`s made the same for your language version?

Reply:
Ah, i let this pass. OK is the right one